Updated with python packages example.

rra94 opened the pull request:
<!-- Add the issue number addressed, if relevant, and complete below -->
Closes #

Summary of changes:
 -

---
Checklist
 - [ ] resources used added to _extras/resources.md
 - [ ] I would like to be recognized for my contribution, so I've added myself to AUTHORS
 - [ ] I have used the learner profiles and contributing guidelines to inform my contribution


Please delete the text below before submitting your contribution.


---

Thanks for contributing!  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.   

---

rra94 committed:
Updated with python packages example.

brownsarahm commented on _episodes/03-packaging-installing.md:
```diff
@@ -28,12 +28,73 @@ Pip is used to install packages from FIXME.
 ## Python Packages
 
-A module is an installable unit of code.
+A module is a piece of code that serves a paticular functionalty. In python moduls ecd with a **.py** extension and the name of the module is the name of the file. A module can contain classes, functions or a combination of both. Packages are namespaces which contain multiple packages and modules.
```
can you also define namespace? some of the learners will be novice programmers and most of the the target audience wil be focused on their research and using programming as a tool, so concepts like that aren't fresh in memory

brownsarahm replied:
moduls ecd = modules end ?

brownsarahm commented on _episodes/03-packaging-installing.md:
```diff
@@ -28,12 +28,73 @@ Pip is used to install packages from FIXME.
 
 
-The file `__init__.py`
+Working with Python packages and modules is simple. We need to:
```
remove simple to make it more inclusive and welcoming.

brownsarahm commented on _episodes/03-packaging-installing.md:
````diff
@@ -28,12 +28,73 @@ Pip is used to install packages from FIXME.
+
+### Step 2: Adding Modules
+
+```
````
the syntax for the styles in this repo is:
`~~~` instead of ``` 
and ends with {: .language-python} or {: .language-bash}

brownsarahm commented on _episodes/03-packaging-installing.md:
````diff
@@ -28,12 +28,73 @@ Pip is used to install packages from FIXME.
+
+```
+class Land:
+    def __init__(self):
````
can the example be functions instead of classes they're used more by novice programmers and are included in the material that is listed as prerequisite, but classes are not.

brownsarahm commented on _episodes/03-packaging-installing.md:
```diff
@@ -28,12 +28,73 @@ Pip is used to install packages from FIXME.
+
+### Step 3 Adding the init file
+
+Finally, we create a file named __init__.py inside the **Vehilces** directory and add the following code:
```
spelling vehicles

brownsarahm commented on _episodes/03-packaging-installing.md:
````diff
@@ -28,12 +28,73 @@ Pip is used to install packages from FIXME.
+from Land import Land
+from Water import Water
+```
+and that's all. Now we can import the package **Vehicles** 
````
remove "and that's all" to keep it motivating

brownsarahm reviewed (changes requested):
this is a great contribution overall; i marked a few spelling/syntax errors and requested a couple changes for tone and content, if you can make those I will merge it right away. 

If possible, but not required prior to merging, can you think of an exercise we could give learners to practice this concept after this section of tutorial?

rra94 reviewed (commented):
Okay this seems fair.

rra94 committed:
incorporated all the changes.

Added a function example
Checked Spelling
Removed And That's all and simple
Added what a namespace is
Added an exercise

brownsarahm committed:
exercise rendering

add syntax for the exercise to render as a box.

brownsarahm reviewed (approved):
thanks!
